jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/SsoSessionConfigurationManager.kt

```diff
@@ -21,31 +21,32 @@ import java.util.Optional
 class SsoSessionConfigurationManager {
 
     val profileFile = ProfileFileLocation.configurationFilePath().toFile()
-    private fun writeSsoSessionProfileToConfigFile(
+    fun writeSsoSessionProfileToConfigFile(
+        idcProfileName: String,
```

Comment:
can't we infer the profile name from the existing paramters?

Reply:
Yes, will fix this